### ISSUE TITLE
`require-description` doesn't detect spaced out multiline comments

### ISSUE THREAD

**Issue by ravindUwU** [eslint-community/eslint-plugin-eslint-comments#247]
While `require-description` correctly detects `// eslint-disable-next-line ...` and `/* eslint-disable-next-line ... */` comments, it doesn't seem to work with multiline comments formatted like:

```
/*
	eslint-disable-next-line ...
*/
```

I use this formatting when the description is very long, or project conventions suggest wrapping code at a certain number of characters:

```
/*
	eslint-disable-next-line ... --
	Long description goes here, instead of on the preceding line.
*/
```

Reproduced at: https://stackblitz.com/edit/stackblitz-starters-byqdswjs?file=index.js
